CMakeLists.txt

```diff
@@ -33,6 +33,11 @@ option(IS_FASTDDS_SH_USING_FASTDDS_EXTERNALPROJECT
 
 option(BUILD_LIBRARY "Compile the Fast DDS SystemHandle" ON)
 
+###################################################################################
+# Set Aliases for FastDDS SH
+###################################################################################
+set (ALIASES "databroker")
```

Comment:
If the Integration Core PR is not intended to be merged this must be reverted

Reply:
:+1: